README.md

```diff
@@ -128,6 +128,15 @@ release = PuppetForge::Release.find release_slug
 # @raise PuppetForge::ReleaseNotFound error if the given release does not exist
 release.download(Pathname(release_tarball))
 
+# Upload a module tarball to the Puppet Forge
+# Returns an instannce of V3::Release class and the response from the forge upload 
+# @raise PuppetForge::ReleaseForbidden if a 403 response is recieved from the server
+# @raise PuppetForge::ReleaseBadContent if the module to upload is not valid
+# @raise Faraday::ClientError if any errors encountered in the upload
+# @raise PuppetForge::FileNotFound if the given tarball cannot be found
+PuppetForge::Connection.authorization = 'xxxxx'
```

Comment:
I assume this was left in accidentally and you'll want consumers of this method to set this in their own code, yeah?

Reply:
This is just and example right? My thought was to be explicit that authentication is required.

Reply:
Actually thinking about this in context it does make sense at all.

I think that upload should have its own example section.

Reading through the example in the readme at the moment is confusing.

Will update accordingly.

Reply:
oh, I lost track of which file I was reading, haha. Ignore me :D